profiles/nuvla.json

```diff
@@ -0,0 +1,129 @@
+{
+  "name": "Nuvla",
+  "revision": "2015-10-13",
+  "vendor_verified": "2015-10-13",
+  "url": "http://nuv.la/login",
+  "status": "production",
+  "status_since": "2012-01-01",
+  "type": "Generic",
+  "hosting": {
+    "public": true, "private": true
+  },
+  "pricing": [
+    {
+      "model": "metered",
+      "period": "monthly"
+    },
+    {
+      "model": "free"
+    }
+  ],
+  "scaling": {
+    "vertical": true, "horizontal": true, "auto": false
+  },
+  "runtimes": [
+    { "language": "python",     "versions": [ "*.*" ] },
+    { "language": "ruby",       "versions": [ "*.*" ] },
+    { "language": "bash",       "versions": [ "*.*" ] },
+    { "language": "powershell", "versions": [ "*.*" ] }
```

Comment:
I tend to disagree here. I also think we don't need this here as only services reference these runtimes and they do not need a runtime property.